website/content/specification/syntax/constraints.md

```diff
@@ -179,6 +179,21 @@ A constraint may have an OPTIONAL [`@level`](#level) attribute and/or an OPTIONA
 
 If defined, the `<message>` value MUST be a [Metaschema string value](/specification/datatypes#string). It MAY contain a Metapath expression templates that starts with `{`, contains a Metapath expression, and ends with `}`.  When evaluating a template Metapath expression, the context of the Metapath [evaluation focus](#constraint-processing) MUST be the failing value node.
 
+## `is-unique` constraints
+
+The `is-unique` constraint is a type of Metaschema constraint that restricts field or flag value(s) based on a component not being referenced more than once.
+
+The `id` attribute of an `is-unique` constraint specifies an identifier for the constraint.
+
+The `@target` attribute of an `<is-unique>` constraint specifies the node in a component definition whose value is distinct. The `is-unique` constraint MUST define a [`@target`](#target) with a Metapath expression.
```

Comment:
Not sure how component definitions relate here. This should be consistent with the text used in similar constraints. The target identifies the nodes that are checked for uniqueness based on the computed key.

Reply:
I may have to revisit this comment.